packages/experimental/RadioGroup/src/RadioGroup.types.ts

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { ITextProps } from '@fluentui-react-native/text';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
```

Comment:
We should pull IViewProps from @fluentui-react-native/adapters instead. react-native-win32 is win32 specific.